pkg/k8s/configmaps/crud_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configmaps_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	oktetoErrors "github.com/okteto/okteto/pkg/errors"
+	"github.com/okteto/okteto/pkg/k8s/configmaps"
+	apiv1 "k8s.io/api/core/v1"
+	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+	k8sTesting "k8s.io/client-go/testing"
+)
+
+func TestGet(t *testing.T) {
+	ctx := context.Background()
+	ns := "fake-ns"
+	mockCfgName := "fake-cfg"
+	mockCfgStatus := "fake-status"
+	mockCfg := &apiv1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      mockCfgName,
+			Namespace: ns,
+		},
+		Data: map[string]string{
+			"status": mockCfgStatus,
+		},
+	}
+	clientset := fake.NewSimpleClientset(mockCfg)
+
+	cfg, err := configmaps.Get(ctx, mockCfgName, ns, clientset)
+	if err != nil {
```

Comment:
NIT: We usually use https://github.com/stretchr/testify to tests the code. There is no need to have lots of if statements and it also provides what's the difference between both objects
```suggestion
	require.NoError(t, err)
```